### PR TITLE
Release 2020-01-19

### PR DESCRIPTION
**Pull request by oliche**
- new qc field in Session
- datasets md5 field renamed to hash
- admin interface dataset server status
- rest endpoint register-file new options (filesize, hash, only-server)

### CODE REVIEW

File: alyx/actions/serializers.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ class SessionDatasetsSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Dataset
-        fields = ('id', 'name', 'dataset_type', 'data_url', 'url')
+        fields = ('id', 'name', 'dataset_type', 'data_url', 'url', 'file_size', 'md5')
```
Comment (rossant) [inline alyx/actions/serializers.py]: hash instead of md5?

File: alyx/data/models.py [hunk 2]
```diff
@@ -253,6 +253,15 @@ class Dataset(BaseExperimentalData):
     md5 = models.UUIDField(blank=True, null=True,
```
Comment (rossant) [inline alyx/data/models.py]: we don't delete the md5 field?

**Comment by rossant** [comment]
@oliche I pushed a commit that requires a migration: allowing allowed_users to be blank (needed when adding a new user)

File: alyx/data/models.py (hunk 2 above)
Comment (oliche) [reply alyx/data/models.py]: need to do it as a 2 steps migration because cortex lab has some data in this field.

**Pull request merged**